tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -277,18 +277,36 @@ abstract class BaseWorksheetLspSuite(scalaVersion: String)
       _ = assertNoDiff(
         client.workspaceDiagnostics,
         getExpected(
-          """|a/src/main/scala/Main.worksheet.sc:2:1: error: java.lang.RuntimeException: boom
-             |	at repl.MdocSession$App.<init>(Main.worksheet.sc:11)
+          """|a/src/main/scala/Main.worksheet.sc:2:1: error: java.lang.ExceptionInInitializerError
```

Comment:
Because we changed to an object, the exception stack traces changed, but they are shown in the same place. We could possibly improve them on the mdoc side, but they seem ok for now.